modules/videopress/shortcode.php

```diff
@@ -177,10 +180,10 @@ public function video_shortcode_override($html, $attr, $content, $instance) {
 			$url_keys = array( 'src', 'mp4' );
 
 			foreach ( $url_keys as $key ) {
-				if ( isset ( $attr[ $key ] ) ) {
+				if ( isset( $attr[ $key ] ) ) {
 					$url = $attr[ $key ];
 
-					if ( preg_match( '@videos.(videopress\.com|files\.wordpress\.com)/([a-z0-9]{8})/@i', $url, $matches ) ) {
+					if ( preg_match( '@videos.(videopress\.com|files\.WordPress\.com)/([a-z0-9]{8})/@i', $url, $matches ) ) {
```

Comment:
You will need to add a rule to ignore here, like I did for comments.

Reply:
Resolved.